content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -14,9 +14,15 @@ can specify on different Kubernetes components.
 ## Overview
 
 Feature gates are a set of key=value pairs that describe alpha or experimental
-features.
+features.  
 An administrator can use the `--feature-gates` command line flag on each component
-to turn a feature on or off.
+to turn a feature on or off. Certain features are valid for certain components.  
+For example adding option to `--feature-gates` flag of kubelet to enable feature `DynamicKubeletConfig`:  
+
+```shell
```

Comment:
@charrywanganthony
- possibly re-word, `Certain features are valid for certain components`
- could provide an example, showing how to get the feature list per component?

Each component supports a set of feature gates unique to that component.
To see the list of features for a component, use the `--feature-gates` flag:
```shell
kube-apiserver --feature-gates
```
To set feature gates for a component, use the  `--feature-gates` flag assigned to a list of feature pairs:

```shell
kubelet --feature-gates="...,DynamicKubeletConfig=true"
```

Reply:
Thanks @kbhawkey .
Modified now.